components/FeatureHighlight/src/MDCFeatureHighlightView.h

```diff
@@ -20,7 +20,11 @@
 
 @property(nonatomic, strong, nullable) UIColor *innerHighlightColor UI_APPEARANCE_SELECTOR;
 @property(nonatomic, strong, nullable) UIColor *outerHighlightColor UI_APPEARANCE_SELECTOR;
+
+@property(nonatomic, strong, nullable) UIFont *titleFont UI_APPEARANCE_SELECTOR;
```

Comment:
Added.

Reply:
did you forget to add it to the PR?

Reply:
UI Test added.

If you are specifically talking about a Unit test for titleFont, it is simply wired to a backing _ivar.

Let's catch up offline if there are still open questions.

Reply:
oh I was expecting something run by a test. Are you saying you created a manual example to test?

Reply:
I would like a unit test that checks that the backing label's font changes when you set the public api.

Since we are embracing a "I will set a different font then they one you want" (dynamic text) we should unit test that as well.

Also we should test that the default font gets used when we set the font to nil.